### DISCUSSION TITLE
Release Schedule

### DISCUSSION THREAD

**Discussion by KyleAure** [containers/podman-compose#702] [General]
I wasn't able to find any information about a release schedule for this project.
I think having a consistent release schedule would be good for the project and the podman community in general. 

I specifically want to start using the `-profile` option which was put in right after the 1.0.6 release. 
Without a clear release schedule, I now need to decide if podman/podman-compose is a viable option for my project or am I forced to use docker/docker-compose?

**Comment by Abashinos** [comment]
I second that question.

We use profiles as well, so podman-compose 1.0.6 is unsuitable for production, since it starts all defined services disregarding profile requirements.

Pulling and building from `devel` is obviously a risky approach and the criteria for stability of a given build is unclear beyond a successful test run (I'm guessing for both `pytest` and `test` suites?).
Also, current version in docker_compose.py is set to 1.0.7 without a corresponding Github tag or PyPI build, which only adds to the confusion...

I'm willing to help out with some house cleaning, but I'd like to know how to successfully target a stable version and plan the upgrade accordingly.
